yaml/csr.yaml.m4

```diff
@@ -59,6 +60,11 @@
 #      - type: ...
 #      - ...
 
+
+changequote(`[[[', `]]]')
```

Comment:
Avoid conflict with SystemVerilog syntax and other comments by choosing these special quotation marks.